commands/command_edit.cpp

```diff
@@ -1361,6 +1361,9 @@ void EditSensors::mapSensors(int toCyl, int fromCyl)
 		for (int s = 0; s < MAX_SENSORS; ++s) {
 			if (dc->sample[i].pressure[s].mbar && dc->sample[i].sensor[s] == fromCyl)
 				dc->sample[i].sensor[s] = toCyl;
+			// In case the cylinder we are moving to has a sensor attached, move it to the other cylinder
+			else if (dc->sample[i].pressure[s].mbar && dc->sample[i].sensor[s] == toCyl)
+				dc->sample[i].sensor[s] = fromCyl;
```

Comment:
so if both to and from cylinders had a sensor, then they get swapped, but if no sensor was attached to to cylinder before, we still have the old behavior.
Very clever.

Reply:
Yea, I remember that I thought the swap case felt really messy and then it turned out to be simpler than making the cylinders with sensors non-editable. =)